src/backend/distributed/deparser/deparse_text_search.c

```diff
@@ -305,6 +460,37 @@ DeparseTextSearchConfigurationCommentStmt(Node *node)
 }
 
 
+/*
+ * DeparseTextSearchDictionaryCommentStmt returns the sql statement representing
+ * COMMENT ON TEXT SEARCH DICTIONARY ... IS ...
+ */
+char *
+DeparseTextSearchDictionaryCommentStmt(Node *node)
+{
+	CommentStmt *stmt = castNode(CommentStmt, node);
+	Assert(stmt->objtype == OBJECT_TSDICTIONARY);
+
+	StringInfoData buf = { 0 };
+	initStringInfo(&buf);
+
+	appendStringInfo(&buf, "COMMENT ON TEXT SEARCH DICTIONARY %s IS ",
+					 NameListToQuotedString(castNode(List, stmt->object)));
+
+	if (stmt->comment == NULL)
+	{
+		appendStringInfoString(&buf, "NULL");
```

Comment:
codecov complains about this line not being hit, seems like low hanging fruit to increase the coverage for free.

Mind adding a test that sets the comment to `NULL`?

Reply:
done.